js/script.js

```diff
@@ -0,0 +1,8 @@
+var classButton = document.getElementsByClassName("button");
+        console.log(classButton);
+        var i;
+        var text = "";
+        for (i = 0; i < classButton.length; i++) {
+            text += classButton[i].innerText + "<br>";      
+        }
+        document.getElementById("button_names").innerHTML = text;
```

Comment:
why you add <br> to innerText and then you set it to innerHTML? better if you add it to innerHTML in loop